src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTag.cs

```diff
@@ -64,7 +64,7 @@ internal class InlineDiagnosticsTag : GraphicsTag, IEndOfLineAdornmentTag
         /// <summary>
         /// Creates a GraphicsResult object which is the error block based on the geometry and formatting set for the item.
         /// </summary>
-        public override GraphicsResult GetGraphics(IWpfTextView view, Geometry bounds, TextFormattingRunProperties format)
+        public override GraphicsResult GetGraphics(IWpfTextView view, Geometry unused, TextFormattingRunProperties format)
```

Comment:
Does using a discard make sense here?

```suggestion
        public override GraphicsResult GetGraphics(IWpfTextView view, Geometry _, TextFormattingRunProperties format)
```

Reply:
it uses a discard in a local function down below, so I had to use that identifier instead